cat-launcher/src/components/ui/combobox.tsx

```
@@ -0,0 +1,115 @@
+  }, [autoselect, value, items, onChange]);
+
+  return (
+    <div>
+      {label ? (
+        <div className="text-sm text-muted-foreground mb-2">{label}</div>
+      ) : null}
+      <Popover open={open} onOpenChange={setOpen}>
+        <PopoverTrigger asChild>
+          <Button
+            variant="outline"
+            role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)